internal/provider/meta_test.go

```diff
@@ -530,497 +526,3 @@ func TestIsEnterpriseSupported(t *testing.T) {
 		})
 	}
 }
-
-func TestNewProviderMeta(t *testing.T) {
```

Comment:
In order to determine whether boolean fields are unset or set to false in the SDKv2 provider, we are relying on the `RawConfig` as part of the resource data. This `RawConfig` is unavailable during tests that are not using the TF Testing framework. The following equivalent tests `TestNewProviderMeta` and `TestNewProviderMeta_Cert` are setting up the Provider Meta from scratch, and are not setting up a full TF test that runs a `terraform apply`, which will not be possible going forward (`RawConfig` will always be nil during this test run, causing a panic)

All test-cases below have been manually tested using TF Configs, and have been documented. Please reach out to get a link to the document if you are unable to find it in Slack!